modules/monitoring-managing-user-permissions-using-cli.adoc

```diff
@@ -0,0 +1,40 @@
+// Module included in the following assemblies:
+//
+// * monitoring/monitoring-your-own-services.adoc
+
+[id="managing-user-permissions-using-cli_{context}"]
+= Managing user permissions using CLI
+
+This procedure shows how to manage user permissions for monitoring their own services using the CLI.
+
+.Prerequisites
+
+* You need to have a user created.
+* Log in using the `oc` command.
+
+.Procedure
+
+This example shows how to assign permissions to user `johnsmith` for setting up metrics collection and creating alerting rules in the `ns1` namespace:
```

Comment:
See my notes individually below, but I think this section needs to be a little clearer. I think annotations might help. :)

Reply:
Made several changes to the section to make it less confusing, inspired by your feedback.